componentapiexample/src/main/java/net/gini/android/vision/component/GiniVisionActivity.java

```diff
@@ -429,29 +427,22 @@ protected void onStart() {
         }
     }
 
-    public boolean isShowingCamera() {
-        Fragment fragment = getFragmentManager().findFragmentById(R.id.fragment_container);
-        return fragment != null && fragment instanceof CameraFragmentStandard;
-    }
-
     public void removeOnboarding() {
         LOG.debug("Remove the Onboarding Screen");
         showCameraOverlays();
         Fragment fragment = getFragmentManager().findFragmentById(
                 R.id.fragment_container_onboarding);
         if (fragment != null) {
-            getFragmentManager().beginTransaction()
-                    .setCustomAnimations(R.animator.fade_in, R.animator.fade_out)
-                    .remove(fragment)
-                    .commit();
+            getFragmentManager().beginTransaction().setCustomAnimations(R.animator.fade_in,
```

Comment:
Why did you change the indentation? For me it is more readable in the previous way.

Reply:
It wasn't me (:P), the code style made this. I fixed it now to preserve line breaks.